src/commands/utils/taf.ts

```diff
@@ -0,0 +1,79 @@
+import fetch from 'node-fetch';
+import { Colors } from 'discord.js';
+import { CommandDefinition, replyWithEmbed } from '../../lib/command';
+import { CommandCategory, Units } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import Logger from '../../lib/logger';
+
+export const taf: CommandDefinition = {
+    name: 'taf',
+    description: 'Provides the TAF report of the requested airport',
+    category: CommandCategory.UTILS,
+    executor: async (msg) => {
+        const splitUp = msg.content.replace(/\.taf\s+/, ' ').split(' ');
+
+        if (splitUp.length <= 1) {
+            const noQueryEmbed = makeEmbed({
+                title: 'TAF Error | Missing Query',
+                description: 'You must provide an airport ICAO code.',
+                color: Colors.Red,
+            });
+            await msg.reply({ embeds: [noQueryEmbed] });
+            return Promise.resolve();
+        }
+        const icaoArg = splitUp[1];
+
+        try {
+            const tafReport = await fetch(`https://avwx.rest/api/taf/${icaoArg}`, {
```

Comment:
Should this not use the FBW API that provides caching etc?

Reply:
I'm not sure, just used the same one that was used in the .metar command

Reply:
We could change this but I see no real need to. Vatsim uses or have used AVWX and we've been very kindly given a plan by AVWX for free.

Am open to changing this if we feel the need - I'm sure there was a discussion about this when we originally chose to go with AVWX, not sure though!